### PR TITLE
Defect sysmodel exported

### PR DESCRIPTION
**Pull request by mbwhite**
Have added code to the toArchive to prevent the system model file being included
Also added code comments to the private APIs re their usage

### CODE REVIEW

**Comment by mbwhite** [comment]
Build has completed - but will add more tests into the model manager file.... 
But the fix in and of itself is good to go

File: packages/composer-common/lib/businessnetworkdefinition.js [hunk 1]
```diff
@@ -300,6 +300,10 @@ class BusinessNetworkDefinition {
         zip.file('models/', null, Object.assign({}, options, { dir: true }));
         modelFiles.forEach(function(file) {
             let fileName;
+            // ignore the system namespace when creating an archive
+            if (file.namespace === COMPOSER_SYSTEM_NAMESPACE){
```
Comment (sstone1) [inline packages/composer-common/lib/businessnetworkdefinition.js]: This should be `file.isSystemModelFile()` shouldn't it?

File: packages/composer-common/lib/modelmanager.js [hunk 2]
```diff
@@ -172,6 +172,9 @@ class ModelManager {
         LOG.info(NAME,'updateModelFile',modelFile,fileName);
         if (typeof modelFile === 'string') {
             let m = new ModelFile(this, modelFile, fileName);
+            if (m.getNamespace === ModelUtil.getSystemNamespace()){
```
Comment (sstone1) [inline packages/composer-common/lib/modelmanager.js]: And here, should be `m.isSystemModelFile()` and also you're missing some `()` around `getNamespace` anyway.

File: packages/composer-common/lib/businessnetworkdefinition.js (hunk 1 above)
Comment (mbwhite) [inline packages/composer-common/lib/businessnetworkdefinition.js]: Ah  - useful method - forgetting I'm not in Eclipse anymore...
Yep change it.

File: packages/composer-common/lib/modelmanager.js (hunk 2 above)
Comment (mbwhite) [inline packages/composer-common/lib/modelmanager.js]: changed

**Pull request merged**